cluster/kubernetes/sync.go

```diff
@@ -142,12 +143,17 @@ func (c *Cluster) collectGarbage(
 		actual := res.GetChecksum()
 		expected, ok := checksums[resourceID]
 
+		collectGarbage := !ok && !dryRun
```

Comment:
Michael already mentioned this, but I think adding `dry-run` to the log would be sufficient. E.g.

```go
switch {
case !ok: // was not recorded as having been staged for application
  c.logger.Log("info", "cluster resource not in resources to be synced; deleting", "dry-run", dryRun, "resource", resourceID)
  if !dryRun {
    orphanedResources.stage("delete", res.ResourceID(), "<cluster>", res.IdentifyingBytes())
  }
case actual != expected: 
  c.logger.Log("warning", "resource to be synced has not been updated; skipping", "dry-run", dryRun, "resource", resourceID)
  continue
}
```

Reply:
sure, I can do that. I thought he meant something else.

Reply:
@hiddeco done